packages/app/cypress/e2e/subscriptions/specChange-subscription.cy.ts

```diff
@@ -12,7 +12,8 @@ describe('specChange subscription', () => {
   describe('specs list', () => {
     it('responds to specChange event for an added file', () => {
       cy.get('[data-cy="spec-item-link"]')
-      .should('have.length', 14)
+      // cannot assert a length since this is a virtualized list
```

Comment:
It doesn't, but it makes it a bit more complex, since the number of nodes rendered is determined by the height of the list in the current conte plus the `overscan` number that is configured for the list in the Vue component that renders it. It will sometimes be less than then number of matching results for a given search term.